pkg/sql/doc.go

```diff
@@ -111,16 +116,16 @@ corresponds to the primary index:
   /TableID/PrimaryIndexID/parentID/name
 
 The column data associated with a row in a table is stored within the primary
-index which is the index associated with the primary key. Every column has a
-unique ID (that is local to the table). The value for a column is stored at the
-key:
+index. Additionally every column has a unique ID (local to the table). The
+values for every column is stored at the key:
 
   /TableID/PrimaryIndexID/parentID/name/ColumnID -> Value
 
 A column containing a NULL value is not stored in the monolithic map. In order
 to detect rows which only contain NULL values in non-primary key columns, every
 row has a sentinel key indicating its existence. The sentinel key is simply the
-primary index key:
+primary index key: TODO(Arjun): I do not understand this paragraph, it needs
+clarification.
```

Comment:
Try this:

"""
A column containing a NULL value is simply omitted from the monolithic map; its absence indicates the NULL value.
To cover the case where all the non-PK columns of a row happen to be NULL, a special encoding is used: a "sentinel" key, which is simply the primary index key of that row without any column suffix, is used to indicate the presence of the row.
(FIXME: is the following true?) Sentinel keys are only used for tables that have zero non-null, non-PK columns.
"""